modules/examples/todolist/runtime/src/main/scala/PingPong.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2017-2018 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package examples.todolist.runtime
+
+import cats.effect.IO
+import cats.~>
+import monix.eval.Task
+import monix.execution.Scheduler
+
+trait PingPong {
+
+  implicit val S: Scheduler = Scheduler.Implicits.global
+
+  implicit def T2IO(implicit S: Scheduler): Task ~> IO = new (Task ~> IO) {
+    override def apply[A](fa: Task[A]): IO[A] = fa.toIO
+  }
+
+  implicit def T2Task(implicit S: Scheduler): Task ~> Task = new (Task ~> Task) {
```

Comment:
If we remove the double interpretation this NT could be removed.

Reply:
:+1: